contrib/submit-protocol/src/App/MarketplaceForm.tsx

```diff
@@ -126,6 +131,10 @@ export default class MarketplaceForm extends React.Component<IMarketplaceProps,
                   value={this.state.uriToken}
                   onChange={this.setMarketplaceURIToken}
                 />
+                <Stack gap={20} padding="15px auto 0" horizontalAlign="center" horizontal={true}>
```

Comment:
If marketplace will be a built-in feature and need to use similar style as other components, you can try to use the following css constants:
- Font: FontClassNames
- Color: ColorClassNames
- Spacing
  - Stack: `l1 l2 m s1 s2`
  - Constant value: `const theme = getTheme(); marginLeft = theme.spacing.l1; blahblahblah`

Reply:
Let's refine css styles later in a separate PR.